core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -217,6 +218,7 @@ public PartitionData copy() {
     this.sortOrderId = toCopy.sortOrderId;
     this.dataSequenceNumber = toCopy.dataSequenceNumber;
     this.fileSequenceNumber = toCopy.fileSequenceNumber;
+    this.manifestPath = toCopy.manifestPath;
```

Comment:
Shall we put this next to `fileOrdinal` to match the order of the fields?

Reply:
Done!